mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -38,7 +38,14 @@
  *
  */
 public interface MailboxMapper extends Mapper {
-    
+
+    /**
+     * Create the given {@link Mailbox} to the underlying storage
```

Comment:
The documentation doesn't tell me how Create and Save are different things.

Reply:
Hopefully within a following step we should be able to `s/save/rename/`

Reply:
oh, it finally makes sense

Reply:
@Arsnael could you:

 - 1. add a precondition and it's tests to ensure **save** is always called with a mailboxId ?
 - 2. rename this method to **rename** <3 ?

Reply:
Will do

Reply:
I did the rename, however I prefer rework on rename in an other PR as it breaks a lot of tests and stuff around in the code if we enforce the fact that rename should be called with a mailboxId (well before using one method to create and rename, now distinct)

Reply:
Agree to have that in another PR then